app/src/main/java/org/p2p/wallet/user/repository/prices/TokenPricesRemoteRepository.kt

```diff
@@ -0,0 +1,35 @@
+package org.p2p.wallet.user.repository.prices
+
+import org.p2p.wallet.common.AppFeatureFlags
+import org.p2p.wallet.home.model.TokenPrice
+import org.p2p.wallet.infrastructure.dispatchers.CoroutineDispatchers
+import org.p2p.wallet.user.repository.prices.sources.CoinGeckoApiClient
+import org.p2p.wallet.user.repository.prices.sources.CryptoCompareApiClient
+import kotlinx.coroutines.withContext
+
+class TokenPricesRemoteRepository(
+    private val cryptoCompareApiClient: CryptoCompareApiClient,
```

Comment:
i prefer to use same interface and provide it in TokenPricesRemoteRepository according to appFeatureFlags.useCoinGeckoForPrices

contact me for mode details if it is not clear

Reply:
do you want to keep only TokenPricesRemoteRepository in this way or just inject one interface and choose repo implementation depending on feature flag?

Reply:
2ed point
to pass interface and choose repo implementation depending on feature flag